server/index.js

```diff
@@ -1,45 +1,112 @@
 import serverless from 'serverless-http';
 import express from 'express';
 import helmet from 'helmet';
+import cors from 'cors';
+import passport from 'passport';
+import TwitterStrategy from 'passport-twitter';
+import session from 'express-session';
+import ConnectMongo from 'connect-mongo';
+import { getMongooseConnection } from './src/store';
+const MongoStore = ConnectMongo(session);
+
 import { getBit, getBits, getComments, createBit } from './src/bits';
 
+passport.use(
+  new TwitterStrategy(
+    {
+      consumerKey: `${process.env.TWITTER_API_KEY}`,
+      consumerSecret: `${process.env.TWITTER_API_SECRET_KEY}`,
+      callback: `${process.env.BASE_SERVER_URL}/oauth/twitter/callback`,
+    },
+    function (token, tokenSecret, profile, cb) {
+      return cb(null, profile);
+    }
+  )
+);
+
+var _requestSecret;
+
 const app = express();
 app.use(express.json()); // support encoded JSON
 app.use(express.urlencoded({ extended: true })); // support encoded bodies
 // Security middleware, but keep client-side caching.
-app.use(helmet({
-  noCache: {
-    action: 'deny'
-  }
-}));
+app.use(
+  helmet({
+    noCache: {
+      action: 'deny',
+    },
+  })
+);
+app.use(
+  session({
+    // TODO(thenuge): Use a better secret.
+    secret: 'kirby',

```

Comment:
I asked TGA to tell me a secret:

i-think-tea-is-overrated-i-mean-its-just-leaves